apps/comps/providers/session-provider.tsx

```diff
@@ -84,6 +85,7 @@ export function SessionProvider({ children }: { children: React.ReactNode }) {
 
   const { user, ready, authenticated, logout, isModalOpen, createWallet } =
     usePrivy();
+  const { ready: walletsReady } = useWallets();
```

Comment:
[nit] i'd prefer to _not_ add another hook into the privy equation, purely for logging purposes.